### PR TITLE
feat: add percentage option #121

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#121

### CODE REVIEW

File: src/DOMRenderer.ts [hunk 1]
```diff
@@ -291,4 +293,24 @@ export default class DOMRenderer {
...
+		const sizePercentage = this._sizePercentage;
+		const posPercentage = this._posPercentage;
+		const viewportSize = this.getViewportSize();
+		const horizontal = this.options.horizontal;
+
+		properties.forEach(p => {
+			if (!(p in rect)) {
+				return;
+			}
+			const horizontalPercentage
```
Comment (mixed) [inline src/DOMRenderer.ts:306]: How about adding the prefix of `is`?

File: src/DOMRenderer.ts [hunk 2]
```diff
@@ -191,9 +153,44 @@ export default class DOMRenderer {
...
+		});
+	}
+	public renderItem(item: IInfiniteGridItem, rect: IInfiniteGridItem["rect"], transitionDuration?: number) {
+		if (!item.el) {
+			return;
+		}
+		const { el, prevRect } = item;
+		const style = el.style;
+
+		// for debugging
```
Comment (mixed) [inline src/DOMRenderer.ts:171]: for debugging? What about debugging?

**Review by mixed — APPROVED** [review]
LGTM

File: src/DOMRenderer.ts (hunk 1 above)
Comment (daybrush) [reply src/DOMRenderer.ts:306]: @mixed 

Good. I will add the `is` prefix.

**Review by WoodNeck — APPROVED** [review]
LGTM

**Pull request merged**